src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import Link from '../../StyledComponents/Link/Link';
+import { CART } from '../../StyledComponents/Link/utils';
+import NavBar from '../../StyledComponents/NavBar/NavBar';
+import { GiHamburgerMenu } from 'react-icons/gi'
+import './Header.scss'
+import UserWindow from '../../utils/hooks/UserWindow';
+
+export const TYPE_LINK = 'link'
+export const TYPE_INPUT = 'input'
+
+const itemsMenuList = [
+    {
+        id: 1,
+        title: 'Home',
+        idItem: 'home',
+        url: '#',
+        type: TYPE_LINK,
+    },
+    {
+        id: 2,
+        title: 'search',
+        idItem: 'search',
+        url: '#',
+        type: TYPE_INPUT,
+        icon: 'search'
+    },
+    {
+        id: 3,
+        title: 'cart',
+        idItem: 'cart',
+        url: '#',
+        type: TYPE_LINK,
+        icon: CART,
+        isIcon: true
+    },
+]
+
+const Header = () => {
+    const [showNavBar, setShowNavBar] = useState(false)
+    const { windowWidth } = UserWindow()
+    const handleClickIconBurger = () => {
+        setShowNavBar(!showNavBar)
+    }
+    const isMobile = () => {
```

Comment:
You can have this on your UserWindow hook or in a utils file so it can be shared across the app